src/extension/background-script/connectors/lnbits.ts

```diff
@@ -17,7 +17,19 @@ import Connector, {
   VerifyMessageResponse,
 } from "./connector.interface";
 
+interface Config {
+  adminkey: string;
+  url: string;
+}
+
 class LnBits extends Base implements Connector {
+  config: Config;
+
+  constructor(config: Config) {
+    super();
```

Comment:
should this always be called? even if Base does not implement it?
I am even thinking if we should completely get rid of `Base` right now nothing is left. 
an `init()` and `unload()` might be needed for future connections (like #444 ) but then those can also just be added to any connector individually.